dipy/align/tests/test_imaffine.py

```diff
@@ -467,7 +482,8 @@ def test_affine_map():
             expected_nn = oracle_nn(img, dom_shape[:dim], aff_inv_inv)
 
             affine_map = imaffine.AffineMap(aff_inv,
-                                            cod_shape[:dim], codomain_grid2world,
+                                            cod_shape[
+                                                :dim], codomain_grid2world,
```

Comment:
I prefer not to break in the middle of an indexing operation. Can you break this line after the comma?